azure-pipelines.yml

```
@@ -1,7 +1,7 @@
 jobs:
   - job: docker_build
     pool:
-      vmImage: ubuntu-20.04
+      vmImage: ubuntu-latest
```

```suggestion
      vmImage: ubuntu-24.04
```

I feel like I often get issue with the C library when i'm on bleeding edge...

---

-latest defaults to 22.04 so far :) See https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software

---

Ah ok